src/run/port.rs

```diff
@@ -128,10 +140,22 @@ impl Port {
     Port::new(Var, 0, addr)
   }
 
-  /// Creates a new [`Num`] port with a given 60-bit numeric value.
+  /// Creates a new [`Int`] port with a given 60-bit numeric value.
+  #[inline(always)]
+  pub const fn new_int(val: i64) -> Self {
+    Port((val << 4) as u64 | (Int as u64))
+  }
+
+  /// Creates a new [`F32`] port with a given 60-bit numeric value.
+  #[inline(always)]
+  pub const fn new_float(val: f32) -> Self {
+    Port((unsafe { std::mem::transmute::<_, u32>(val) as u64 } << 4) | (F32 as u64))
+  }
+
+  /// Creates a new [`Int` | `F32`] port with a given 60-bit numeric value.
```

Comment:
```suggestion
  /// Creates a new [`Int`] or [`F32`] port with a given 60-bit numeric value.
```

(the brackets are rustdoc syntax for links)